src/frontend-pwa/src/views/Settings/Settings.tsx

```diff
@@ -254,38 +316,77 @@ export default function Settings() {
         </Section>
         <Accordion
           content={(
-            <Button
-              handleClick={handleRefresh}
-              variant="primary"
-              size="sm"
-              disabled={!onlineCheck}
-              text={!onlineCheck ? 'Offline' : 'Refresh'}
-            />
+            <div>
+              <div style={{ display: 'flex', gap: '10px' }}>
+                <Button
+                  handleClick={handleRefresh}
+                  variant="primary"
+                  size="sm"
+                  disabled={!onlineCheck}
+                  text={!onlineCheck ? SettingsContent.refreshDataButtonTextOffline[lang] : SettingsContent.refreshDataButtonTextRefresh[lang]}
+                />
+              </div>
+            </div>
           )}
-          text={(SettingsContent.refreshData[lang])}
-          tooltip={(
-            <MoreInfoButton
-              tip={SettingsContent.refreshDataToolTip[lang]}
-            />
+          text={SettingsContent.refreshData[lang]}
+          tooltip={<MoreInfoButton tip={SettingsContent.refreshDataToolTip[lang]} />}
+        />
+        <Accordion
+          content={(
+            <div>
+              <div style={{ display: 'flex', gap: '10px' }}>
+                {state.mapsCached ? (
+                  <Button
+                    handleClick={handleClearCacheConfirm}
+                    variant="secondary"
+                    size="sm"
+                    disabled={!state.mapsCached}
+                    text={SettingsContent.clearCache[lang]}
+                  />
+                ) : (
+                  <Button
+                    handleClick={handleMapTiles}
+                    variant="primary"
+                    size="sm"
+                    disabled={!onlineCheck}
+                    text={!onlineCheck ? SettingsContent.refreshDataButtonTextOffline[lang] : SettingsContent.refreshDataButtonTextRefresh[lang]}
+                  />
+                )}
+                {isClearCacheModalOpen && (
+                  <ModalWrapper>
+                    <p>{SettingsContent.clearCacheConfirmText[lang]}</p>
+                    <div style={{ display: 'flex', gap: '10px', marginTop: '20px' }}>
```

Comment:
We shouldn't be using inline styling, we have the `settings.styles.ts` folder for making styled components